launch/launch/logging/__init__.py

```diff
@@ -300,33 +315,32 @@ def set_log_format(self, log_format, *, log_style=None):
         else:
             self.file_formatter = None
 
-    def get_log_file_path(self, file_name='launch.log'):
+    def get_log_file_path(self, log_file_name='launch.log'):
         """
         Get the absolute path to the given log file.
 
-        :param: file_name of the log file from which to get the absolute path.
+        :param log_file_name: name of the file for which we want full path
         :return: the absolute path to the log file.
         """
-        return os.path.join(self.log_dir, file_name)
+        return os.path.join(self.log_dir, log_file_name)
 
-    def get_log_file_handler(self, file_name='launch.log'):
+    def get_log_file_handler(self):
```

Comment:
same here. also corresponding code including test needs to roll back as original.